.gitignore

```diff
@@ -0,0 +1,2 @@
+# editor files
+.idea/
```

Comment:
this should be part of your [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) as it will happen in too many repositories to count

Reply:
Good idea. I've removed this file from the project.